UIKitInstagram/UIKitInstagram/CommentModel.swift

```diff
@@ -9,6 +9,9 @@ import Foundation
 
 /// Модель комментариев
 struct Comments {
-    var lineOne: String
-    var lineTwo: String
+    var textLine: String
```

Comment:
```suggestion
    var lineText: String
```
в конце пишет то чем является в данном случае Text Пожалуйста запомни уже! Я устал одно и тоже замечание делать